src/app/containers/ArticleFigure/index.jsx

```diff
@@ -35,6 +35,7 @@ const ArticleFigure = ({
 }) => {
   const imageSpan = {
     default: '6',
```

Comment:
This indicates to me that the `default` prop is not actually doing anything...? Is that correct?